porcupine/settings.py

```diff
@@ -841,7 +841,18 @@ def settings_to_gui(junk: object = None) -> None:
         if panedwindow["orient"] == "horizontal":
             panedwindow.paneconfig(pane, width=global_settings.get(option_name, int))
         else:
-            panedwindow.paneconfig(pane, height=global_settings.get(option_name, int))
+            """Use previous height as long as it's not less than two lines high,
+            otherwise set height to two lines high.
+            """
```

Comment:
Nit: `#` should be used for comments unless it's in the beginning of a function, class, or file.

Reply:
👍 What about when the comment is too long for one line?
Use two `#`s?

Reply:
Yes, use multiple `#`s.